MeshPipe/MeshPipe.m

```diff
@@ -64,36 +80,39 @@ - (instancetype)initWithBasePort:(int)basePort count:(int)count peerName:(NSStri
 	_potentialPeers = [NSMutableArray new];
 	_peers = [NSMutableSet new];
 	
-	_listenSocket = [[AsyncUdpSocket alloc] initWithDelegate:self];
-
+	_listenSocket = [[GCDAsyncUdpSocket alloc] initWithDelegate:self
+                                                  delegateQueue:dispatch_get_main_queue()];
 	for(int i = 0; i < count; i++) {
 		int port = _basePort + i;
-		BOOL success = [_listenSocket bindToAddress:@"localhost" port:port error:NULL];
+        NSData *address = [MeshPipe _addressFor:MeshPipeLocalAddress port:port];
+		BOOL success = [_listenSocket bindToAddress:address error:NULL];
```

Comment:
It can bind to port though. And it does allow to connect to string/port combination. So either I can bind all of them to port + string/port, or do `NSData` addresses everywhere. (for sake of coherence)